packages/api-elements/lib/elements/Resource.js

```diff
@@ -59,6 +72,16 @@ class Resource extends ArrayElement {
   get dataStructure() {
     return this.children.filter(item => item.element === 'dataStructure').first;
   }
+
+  /**
+   * @name host
+   * @type StringElement
+   * @memberof ParseResult.prototype
+   */
+  get host() {
+    return this.children
+      .filter(item => item.classes.contains('host'));
+  }
```

Comment:
```suggestion
```

I don't think we need this accessor, according to the design there won't be any hosts inside the content, only inside the `attributes` (handled above).

Reply:
ok, changed